test/latest/util.js

```diff
@@ -19,17 +19,19 @@ async function generate(file, options) {
     args += value;
   }
   // this cat filePath - the dash is the last pipe op
+  const inputStr = `${latestDate}\n\n{"hello": "world"}`
   const httpMessage =
-    `echo -e '${latestDate}\n\n{"hello": "world"}' | cat ${filePath} - | `;
+    `cat ${filePath} - | `;
   const binaryOps = `${options.generator} ${options.command} `;
   const command = httpMessage + binaryOps + args;
-  const result = await runTest(command);
+  const result = await runTest(command, inputStr);
   return result;
 }
 
-function runTest(command) {
+function runTest(command, inputStr) {
   return new Promise((resolve, reject) => {
     const child = exec(command);
+    child.stdin.end(inputStr)
```

Comment:
Hey thank you for using `child.stdin.end` that might actually be what we are looking for. I am approving this, but there are 2 small eslint errors in `util.js` that could be corrected with `eslint --fix`, but if you are busy we can merge this pending another implementation can prove this PR did not break existing functionality (I'm pretty confident it won't).